config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```
@@ -422,6 +422,68 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-e2e-capz-windows-extension
+  - name: pull-e2e-capz-windows-2022-extension-community
+    decorate: true
```

```suggestion
    cluster: k8s-infra-prow-build
    decorate: true
```

so it will run on community infa